.changeset/93e02260/changes.json

```diff
@@ -0,0 +1,15 @@
+{
+  "releases": [{ "name": "@keystone-alpha/passport-auth", "type": "major" }],
+  "dependents": [
+    {
+      "name": "@keystone-alpha/cypress-project-social-login",
+      "type": "patch",
+      "dependencies": ["@keystone-alpha/passport-auth"]
+    },
+    {
+      "name": "@keystone-alpha/cypress-project-twitter-login",
```

Comment:
Since `cypress-project-twitter-login` has been removed, we should remove this entry from the `dependents` list here.

Reply:
I am consolidating this with changes for keystone package.